phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/engines/ViewEngines.java

```diff
@@ -90,7 +90,7 @@ public ViewEngines(ThemeManager themeManager, String enginesPath) {
 
         Bindings.bindContent(availableEngines.getTabs(), mappedSubCategoryTabs);
 
-        this.initializeSidebar();
+        this.setSidebar(this.sidebar);
```

Comment:
I think that it may be better to do it the other way around. I.e. we remove the `setSidebar` method and only provide the `initializeSidebar` method. The reason for this is, that I think we don't want to make the sidebar swapable from outside. In addition a setter method is in my eyes normally not used to directly set complete graphic components. This is normally done by `populate` or `initialize` methods.